builders/integrations/indexers/subquery.md

```diff
@@ -50,216 +53,332 @@ In general, you will need to:
 
 3. You'll be prompted to answer a series of questions:
 
-    1. For the **Select a network family** question, you can choose **Substrate**
+    1. For the **Select a network family** question, although Moonbeam is EVM compatible, the Moonbeam templates are under the **Polkadot** family, so you can choose **Polkadot**
 
-        ![Select Moonbeam](/images/builders/integrations/indexers/subquery/subquery-1.png)
+        ![The SubQuery CLI choose a network family menu.](/images/builders/integrations/indexers/subquery/new/subquery-1.png)
 
-    2. The next screen will prompt you to **Select a network**. At time of writing this guide, Moonriver was the only option. You can go ahead and choose **Moonriver**, and it can be adapted for Moonbeam or Moonbase Alpha
+    2. The next screen will prompt you to **Select a network**. At the time of writing this guide, Moonbeam and Moonriver were the only options
```

Comment:
Were the only 2 options seems a bit negative. I would prefer something like "You can choose between Moonbeam and Moonriver"